### PR TITLE
Fix content-type for fallback static file

### PR DESCRIPTION
**Pull request by snobu**
Fixes https://github.com/falconry/falcon/issues/1313

### CODE REVIEW

File: falcon/routing/static.py [hunk 1]
```diff
@@ -96,16 +95,21 @@ def __call__(self, req, resp):
...
         except IOError:
             if self._fallback_filename is None:
                 raise falcon.HTTPNotFound()
             try:
                 resp.stream = io.open(self._fallback_filename, 'rb')
+                serve_fallback_file = True
             except IOError:
                 raise falcon.HTTPNotFound()
 
+        if (serve_fallback_file):
```
Comment (kgriffs) [inline falcon/routing/static.py]: Nitpick: It would probably be more pythonic to remove the parens here.

**Comment by vytas7** [comment]
Would it be possible to add some sort of regression test, for instance, a new unit test that would have been failing with the old code?

**Comment by vytas7** [comment]
The new PR reflecting the joint effort is out: https://github.com/falconry/falcon/pull/1348

Thanks to everyone involved for the genuine effort in fixing this bug! I have merely put the pieces together and added some unit tests & documentation notes.

**Comment by kgriffs** [comment]
[superseded by  #1348]

**Pull request closed without merging**